### PR TITLE
feat(common): added alerts on text click

### PR DESCRIPTION
**Pull request by meet-shekhar2009**
added the event handlers to every greeting text list, whenever the user will click the text same text will be shown as alert

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,8 +9,14 @@
...
+  function show(event) {
+    debugger;
+    alert(event.innerText);
+  }
+
   const arr = new Array(20).fill(0).map((_, i) => i + 1);
   arr.forEach((element) => {
-    document.body.innerHTML += '<div>Hello user ' + element + '</div>';
+    document.body.innerHTML +=
+      '<div onclick="show(this)">Hello user ' + element + '</div>';
```
Comment (mishranutan) [inline index.html:20]: use string literals for value concatenation.

File: index.html [hunk 2]
```diff
@@ -9,8 +9,14 @@
 </html>
 
 <script>
+  function show(event) {
```
Comment (mishranutan) [inline index.html:12]: function name should be meaningful

File: index.html [hunk 3]
```diff
@@ -9,8 +9,14 @@
...
+    debugger;
+    alert(event.innerText);
+  }
+
   const arr = new Array(20).fill(0).map((_, i) => i + 1);
   arr.forEach((element) => {
```
Comment (mishranutan) [inline index.html:18]: you should use for loop here

Comment (meet-shekhar2009) [reply index.html:17]: I think foreach is fine here

**Review by mishranutan — COMMENTED** [review]
looks goo to me +2

**Pull request merged**